SilMock/tests/Google/Service/Directory/Resource/GroupsTest.php

```diff
@@ -13,13 +14,31 @@ class GroupsTest extends TestCase
     // They are very dependent on order run.
     // groups.insert, groups.listGroups, members.insert, members.listMembers
     public string $dataFile = DATAFILE5;
-    public const GROUP_EMAIL_ADDRESS = 'sample_group@groups.example.com';
+    public const GROUP_EMAIL_ADDRESS = 'sample_group@example.com';
+    public const GROUP_ALIAS_ADDRESS = 'ma_org_sample_group@groups.example.com';
+
+    public function testInitialSetup()
+    {
+        $mockGoogleServiceDirectory = new GoogleMock_Directory('anyclient', $this->dataFile);
+        $group = $mockGoogleServiceDirectory->groups->get(self::GROUP_EMAIL_ADDRESS);
+        if ($group !== null) {
+            $mockGoogleServiceDirectory->groups->delete(self::GROUP_EMAIL_ADDRESS);
+        }
+        $group = $mockGoogleServiceDirectory->groups->get(self::GROUP_EMAIL_ADDRESS);
+        self::assertNull($group, "Failed to clean up previous insert tests.");
+        $group = $mockGoogleServiceDirectory->groups->get(self::GROUP_EMAIL_ADDRESS . 'update');
+        if ($group !== null) {
+            $mockGoogleServiceDirectory->groups->delete(self::GROUP_EMAIL_ADDRESS . 'update');
+        }
+        $group = $mockGoogleServiceDirectory->groups->get(self::GROUP_EMAIL_ADDRESS . 'update');
+        self::assertNull($group, "Failed to clean up previous update tests.");
+    }
 
     public function testInsert()
     {
         $group = new GoogleDirectory_Group();
         $group->setEmail(self::GROUP_EMAIL_ADDRESS);
-        $group->setAliases([]);
+        $group->setAliases([self::GROUP_ALIAS_ADDRESS]); // read-only, should not save anything
```

Comment:
Can you help me understand this comment? It seems like it is saying that the `setAliases()` function call is actually read-only, but I don't understand how a "set...()" function call could just be reading data, not changing it.

Reply:
Ahh, as I read the rest of the function I think I understand.

It might be helpful to phrase it something like "aliases will be ignored when calling `insert()`" or something like that. Is that the intent of that comment?

Reply:
Tweaking comments.